Modules/Loadable/Markups/MRML/vtkMRMLMarkupsCurveNode.cxx

```diff
@@ -305,15 +305,22 @@ double vtkMRMLMarkupsCurveNode::GetCurveLength(vtkPoints* curvePoints, bool clos
     {
     return 0.0;
     }
+  if (startCurvePointIndex >= curvePoints->GetNumberOfPoints() + closedCurve)
```

Comment:
I know this is correct but making things more explicit would improve readability:
```suggestion
  // In case of closed curve we allow one more point beyond the curve points
  // (corresponding to the closing segment).
  if (startCurvePointIndex >= curvePoints->GetNumberOfPoints() + (closedCurve ? 1 : 0))
```